weapons.yaml

```diff
@@ -597,6 +597,20 @@ MammothTusk:
 		ImpactSound: gexpwasa.wav
 		ValidImpactTypes: Water
 
+Maverick:
+	ReloadDelay: 10
+	Range: 6c0
+	Report: vintatta.wav
+	ValidTargets: Ground
+	Burst: 2
```

Comment:
What are you aiming for here? These weapons had no burst (also the warhead is missing pretty much).